lib/onc_certification_g10_test_kit.rb

```diff
@@ -78,20 +79,29 @@ class G10CertificationSuite < Inferno::TestSuite
       G10Options::US_CORE_6_REQUIREMENT
 
     ].each do |us_core_version_requirement|
-      validator :default, required_suite_options: us_core_version_requirement do
+      fhir_resource_validator :default, required_suite_options: us_core_version_requirement do
         url ENV.fetch('G10_VALIDATOR_URL', 'http://validator_service:4567')
 
-        us_core_message_filters =
-          case (us_core_version_requirement[:us_core_version])
-          when G10Options::US_CORE_3
-            USCoreTestKit::USCoreV311::USCoreTestSuite::VALIDATION_MESSAGE_FILTERS
-          when G10Options::US_CORE_4
-            USCoreTestKit::USCoreV400::USCoreTestSuite::VALIDATION_MESSAGE_FILTERS
-          when G10Options::US_CORE_5
-            USCoreTestKit::USCoreV501::USCoreTestSuite::VALIDATION_MESSAGE_FILTERS
-          when G10Options::US_CORE_6
-            USCoreTestKit::USCoreV610::USCoreTestSuite::VALIDATION_MESSAGE_FILTERS
-          end
+        cli_context do
+          txServer nil
+          displayWarnings true
+          disableDefaultResourceFetcher true
+        end
+
+        case (us_core_version_requirement[:us_core_version])
+        when G10Options::US_CORE_3
+          igs('hl7.fhir.us.core#3.1.1')
+          us_core_message_filters = USCoreTestKit::USCoreV311::USCoreTestSuite::VALIDATION_MESSAGE_FILTERS
```

Comment:
I don't understand why this assignment was changed.

Reply:
It just felt strange to have the `igs` line in the middle of the `case` when the result of that case is assigning to a separate variable. I'm probably going to revert this anyway because I suspect there's a better way to define the IG for each option.

Reply:
Having the `igs` lines there wouldn't interfere with the original assignment style.